sys/syz-extract/fuchsia.go

```diff
@@ -25,18 +25,22 @@ func (*fuchsia) prepareArch(arch *Arch) error {
 }
 
 func (*fuchsia) processFile(arch *Arch, info *compiler.ConstInfo) (map[string]uint64, map[string]bool, error) {
-	dir := arch.sourceDir
-	headerArch := arch.target.KernelHeaderArch
-	cc := filepath.Join(dir, "prebuilt", "third_party", "clang", "linux-x64", "bin", "clang")
-	includeDir := filepath.Join(dir, "out", headerArch, "sdk", "exported",
-		"zircon_sysroot", "arch", headerArch, "sysroot", "include")
-	args := []string{"-fmessage-length=0", "-I" + includeDir}
+	srcDir := arch.sourceDir
+	outDir := filepath.Join(srcDir, "out", arch.target.KernelHeaderArch)
+
+	cc := filepath.Join(srcDir, "prebuilt/third_party/clang/linux-x64/bin/clang")
+	args := []string{
+		"-fmessage-length=0",
+		"-I", filepath.Join(outDir, "zircon_toolchain/obj/zircon/public/sysroot/sysroot/include"),
```

Comment:
Ideally this is replaced by sys/targets.Target.CCompiler/CFlags. I think we already have all of that there and that version is better tested and kept up-to-date.

Reply:
Ack. I starting making them more similar, but for some reason using `--sysroot` here (like how it's used in Target.CFlags) led to linking errors, so I stopped and focused on just cleaning up the code as-is for this CL.